About/index.php

```diff
@@ -21,4 +21,6 @@
 </div>
 <?php include("../Global/editUserModal.php"); ?>
 </body>
+<p id="navBarActive" hidden>aboutPage</p>
+<script type="text/javascript" src="../Script/navBar.js"></script>

```

Comment:
JavaScript only need to reference on one page. This is why global folder existence. But this in the bottom of the NavBar.php

Reply:
Test